src/OrchardCore/OrchardCore/Modules/ModularTenantRouterMiddleware.cs

```diff
@@ -77,9 +81,9 @@ public async Task Invoke(HttpContext httpContext)
         }
 
         // Build the middleware pipeline for the current tenant
-        public RequestDelegate BuildTenantPipeline(IServiceProvider rootServiceProvider, IServiceProvider scopeServiceProvider)
+        private RequestDelegate BuildTenantPipeline(IServiceProvider rootServiceProvider, IServiceProvider scopeServiceProvider)
         {
-            var appBuilder = new ApplicationBuilder(rootServiceProvider);
+            var appBuilder = new ApplicationBuilder(rootServiceProvider, _features);
```

Comment:
@jtkech FYI, this change was required to ensure the HTTPS redirection middleware doesn't null-ref because of a null server features collection (which was not populated prior to this change): https://github.com/aspnet/BasicMiddleware/blob/master/src/Microsoft.AspNetCore.HttpsPolicy/HttpsRedirectionBuilderExtensions.cs#L28

Reply:
@PinpointTownes thanks for the info, i approve your change